baas.rst

```diff
@@ -0,0 +1,131 @@
+Backup as a Service
+===================
+
+.. admonition:: Beta Warning
+    :class: note
+
+    This service is currently in beta and seeks for feedback
+
+.. contents::
+
+What is Backup as a Service?
+----------------------------
+
+On APPUiO we provide a managed backup service based on `Restic <https://restic.readthedocs.io/>`__.
+
+It's easy as creating a ``backup`` object in the namespace where the data
```

Comment:
This sentence is somewhat confusing. How about:

Just create a backup object in the namespace you'd like to backup. It's that easy.